docs/installing.rst

```diff
@@ -21,7 +19,11 @@ Installing with get-pip.py
 --------------------------
 
 To install pip, securely download `get-pip.py
-<https://bootstrap.pypa.io/get-pip.py>`_. [2]_
+<https://bootstrap.pypa.io/get-pip.py>`_. [2]_ i.e. 
+
+::
+
+ curl https://bootstrap.pypa.io/get-pip.py -o - | python
```

Comment:
What would be the best practice for downloading and inspecting? Downloading and inspecting are pretty ambiguous instructions.